### PR TITLE
CR for 20399 - Fix near field clipping on avatars with large heads

### PR DESCRIPTION
**Pull request by ctrlaltdavid**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 7b54443bda95a8622a7306225f82c6b47d13610c available at: https://gist.github.com/dc4437a449e0f9ffb58d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4468.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4468.dmg

File: interface/src/avatar/MyAvatar.cpp [hunk 1]
```diff
@@ -1028,7 +1028,28 @@ void MyAvatar::renderBody(ViewFrustum* renderFrustum, RenderMode renderMode, boo
     if (!(_skeletonModel.isRenderable() && getHead()->getFaceModel().isRenderable())) {
         return; // wait until both models are loaded
     }
-    
+
+    Camera *camera = Application::getInstance()->getCamera();
```
Comment (jherico) [inline interface/src/avatar/MyAvatar.cpp]: Technically I think that if the renderMode is MIRROR_RENDER_MODE then you might want to be fetching the Application::_mirrorCamera, although that's not exposed via a getter.

Comment (ctrlaltdavid) [inline interface/src/avatar/MyAvatar.cpp]: If renderMode == MIRROR_RENDER_MODE, getCamera() still returns a valid camera:
- Full body mirror: "mirror" camera
- Rear view mirror: "first person" camera
And both have valid camera positions.
So no code change needed.

**Comment by hifi-gustavo** [comment]
Build console output for commit: c114442325e5bab1ac8592966659f65eb357d405 available at: https://gist.github.com/dc4437a449e0f9ffb58d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4468.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4468.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: c114442325e5bab1ac8592966659f65eb357d405 available at: https://gist.github.com/dc4437a449e0f9ffb58d<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4468.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4468.dmg

**Pull request merged**